assets/src/api.ts

```diff
@@ -187,7 +187,6 @@ export const fetchSwings = (routeIds: RouteId[]): Promise<Swing[] | null> =>
     url: `/api/swings?route_ids=${routeIds.join(",")}`,
     dataStruct: array(SwingData),
     parser: nullableParser(swingsFromData),
-    defaultResult: null,
```

Comment:
question (non-blocking): To make sure I understand the reasoning for this removal, is it because `defaultResult` affected how the parsing error presented?

Reply:
Yeah, if `defaultResult` is provided and there's an error it just provides the default and doesn't throw an exception. Here the default result isn't actually useful so it doesn't improve things versus the case where an exception is thrown, and this way we'll see console output and / or a Sentry alert if there's an issue.